multiversx_sdk/abi/array_value_test.py

```diff
@@ -0,0 +1,71 @@
+from types import SimpleNamespace
+
+import pytest
+
+from multiversx_sdk.abi.array_value import ArrayValue
+from multiversx_sdk.abi.biguint_value import BigUIntValue
+from multiversx_sdk.abi.fields import Field
+from multiversx_sdk.abi.small_int_values import U32Value
+from multiversx_sdk.abi.struct_value import StructValue
+
+
```

Comment:
For the new `ArrayValue` class, let's add some unit tests for encoding / decoding, as well, similar to the ones in Go:

https://github.com/multiversx/mx-sdk-abi-go/blob/main/abi/listValue_test.go#L10

Reply:
Added.